conda_pack/core.py

```diff
@@ -3,6 +3,7 @@
 import glob
 import json
 import os
+import pkg_resources
 import re
```

Comment:
This adds a dependency on `setuptools` - is that something that can be assumed? Alternatively, we can bundle cli-32.exe and cli-64.exe the way conda-build does: https://github.com/conda/conda-build/blob/master/conda_build/cli-64.exe

Reply:
That's fine with my use case, I don't know if that works for @jcrist also.

Reply:
Get the impression that `importlib_resources` (included in Python 3.7) replaces some use cases for `pkg_resources`. Would that work in this case or are features being used here that are not available in `importlib_resources`.

Reply:
I'd rather not depend on Python 3.7 here, but are you saying we should attempt to conditionally import the newer module?

Reply:
Right, meant to point out [`importlib_resources`]( https://pypi.org/project/importlib_resources/ ) is the backport, written by the Python core developers (includes Python 2 support and older Python 3 versions). In Python 3.7 it is `importlib.resources`.

Suggesting it as it seems the Python core developers seem pretty intent on getting people to move away from `pkg_resources`. They even wrote a [migration guide]( https://importlib-resources.readthedocs.io/en/latest/migration.html ).

If `conda-pack` is already dependent on `setuptools` or parts of `pkg_resources` that cannot easily be replaced with `importlib_resources`, understand sticking with `setuptools`.